examples/weldon-on-camelyon/data_managers.py

```diff
@@ -0,0 +1,66 @@
+import math
+from pathlib import Path
+from typing import List
+
+import numpy as np
+import torch
+import torch.nn.functional as F  # noqa: N812
+from torch.utils.data import Dataset
+
+# WARNING: the code is not tested on gpu
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
why do this here? wouldn't it be better if the clients send the data they get `.to(devcie)` when executing `train()` or `test()`? The clients can pass that easily, the global evaluation function can do the same.